io.github.pantheon_tweaks.pantheon-tweaks.yml

```diff
@@ -0,0 +1,112 @@
+app-id: io.github.pantheon_tweaks.pantheon-tweaks
+# elementary SDK is not available on Flathub, so use GNOME SDK and install elementary stylesheet and granite as modules
```

Comment:
> can you use fdsdk 23.08 and the baseapp?

I see, but I guess just replacing with BaseApp would not work here, because BaseApp seems to bundle older versions of elementary stylesheet, icons, and granite.

This project [requires granite >= 6.0.0](https://github.com/pantheon-tweaks/pantheon-tweaks/blob/dc9ed1c2ed387fea4d55d733f57ad89f7c38b7ed/meson.build#L53), so changing BaseApp is fine but we need to either:

- update the granite version in BaseApp
- or keep granite 6.2.0 as modules of this manifest while using BaseApp

Reply:
@bbhtt Is there any reason why BaseApp uses older versions of elementary components? I'd like to update BaseApp itself if possible, so that other developers can use it eventually.

Reply:
ideally the elementary SDK would be available, but it seems elementary was less than interested in doing so.

Reply:
Yea someone has to update it but it should be in a different branch probably. I'm not familiar with elementary SDK's branching conventions.

Is 6 juno shipping granite 6.0.0 or is it in 6.1/7.0? If latter it should be merged to a new branch.

Reply:
at that point it might make more sense as part of `shared-modules`

Reply:
> Is 6 juno shipping granite 6.0.0 or is it in 6.1/7.0? If latter it should be merged to a new branch.

granite 6.x is available since elementary SDK 6 which is the first stable version of that SDK. It ships [granite 6.1.0](https://github.com/elementary/flatpak-platform/blob/5006248df25032f59111dab6704e22ecde6209c0/io.elementary.Sdk.json#L126)

Reply:
Yea then I have no idea why it is on 5.x. @bilelmoussaoui seems to maintain that one and probably has the answers.

Reply:
I just didn't have the energy to do so

Reply:
> @bbhtt Is there any reason why BaseApp uses older versions of elementary components? I'd like to update BaseApp itself if possible, so that other developers can use it eventually.

Then I think you can send a PR if you want.

Reply:
just consider that the base app have an older version and I don't think they are compatible.

IMHO the only that make sense is having the elementary runtime on flathub. But what I heard about that was less than enthusiastic.

Reply:
> > @bbhtt Is there any reason why BaseApp uses older versions of elementary components? I'd like to update BaseApp itself if possible, so that other developers can use it eventually.
> 
> Then I think you can send a PR to update it if you want.

Or just bundle everything and deprecate the baseapp

Reply:
> Or just bundle everything and deprecate the baseapp

That might be better solution. Few developers might use the BaseApp if we update it, since most apps made for elementary OS uses the official elementary runtime and is published to the Flatpak repository hosted in elementary's server…

Reply:
If we don't switch to freedesktop 23.08 and the BaseApp, I think everything we need to build and run this project is already bundled in this manifest and no changes is required here.

Reply:
> deprecate the baseapp

We have to alert all the existing apps using it if we go down that route.

Resolving this as this isn't relevant to the submission anymore.